README.md

```diff
@@ -47,13 +47,83 @@ This demo contains below applications.
 
 ### Automation of Applications in OpenShift.
 
-TODO: -The below deployment will be achieved using Jenkins Pipelines CICD approach.
+**Build and deploy with pipelines**
```

Comment:
Please use Markdown formatting. If this goes under the previous header, which was 3rd level, this would be 4th level and start with "####"